adapter/pyproject.toml

```diff
@@ -12,9 +12,28 @@ python = ">=3.7.1, <3.11"
 dbt-core = "1.3.0b2"
 fal = ">=0.6.0"
 
+## snowflake
+snowflake-connector-python = { version = "^2.7.12", extras = ["pandas"], optional = true }
+
+## bigquery
+# We need the 1.5.0 version to support passing our own client.
```

Comment:
ATL: should it be 1.5 then? what are we doing about this?

Reply:
Just to keep it here I guess, but yeah it doesn't work out of the box. I tried to add a git dependency, but seems like poetry doesn't work when you mark something as both coming from git and optional.